signer/src/bitcoin/validation.rs

```diff
@@ -286,6 +316,7 @@ impl BitcoinPreSignRequest {
         btc_ctx: &BitcoinTxContext,
         requests: &'a TxRequestIds,
         signer_state: SignerBtcState,
+        signers_view_of_last_fees: &mut Option<Fees>,
```

Comment:
Hm, I'm not really following why this is `&mut` and then you're clearing it at the end? Is it that you only want this to be provided for the first transaction in the event of chained utxo's in mempool? If so then I would probably change the `for` loop above to use `.enumerate()` and only provide this on the first iteration instead of doing this dance.